N24DCCN121/baitapbuoi7/cau7.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
+
+string removeDuplicates(string str){
+    string result;
+    for(char ch : str){
+        bool isDuplicate = false;
+        for(char cha : result){
```

Comment:
vẫn đúng, nhưng 2 for lồng cho bài này là phí. Bài này có thể xử lý 1 for. Việc cần làm chỉ là thêm 1 mảng đánh dấu rằng kí tự đó đã xuất hiện trước đó hay chưa, xuất hiện rồi thì k in nửa. Nghiên cứu nha